core/src/main/java/cz/xtf/core/openshift/ClusterVersionOpenShiftBinaryPathResolver.java

```diff
@@ -47,10 +47,19 @@ private String determineClientUrl(final ClusterVersionInfo versionInfo) {
     private String getClientUrlBasedOnOcpVersion(final ClusterVersionInfo versionInfo) {
         Objects.requireNonNull(versionInfo);
 
-        if (versionInfo.getOpenshiftVersion().startsWith("3")) {
+        final String openshiftVersion = versionInfo.getOpenshiftVersion();
+        if (openshiftVersion.startsWith("3")) {
             // OpenShift 3
-            return String.format("%s/%s/%s/oc.tar.gz", OCP3_CLIENTS_URL, versionInfo.getOpenshiftVersion(),
-                    getSystemTypeForOCP3());
+            final String systemTypeForOCP3 = getSystemTypeForOCP3();
+            String downloadUrl = String.format("%s/%s/%s/oc.tar.gz", OCP3_CLIENTS_URL, openshiftVersion,
+                    systemTypeForOCP3);
+            // if the generated download URL is not working (404 or 403 response code) try to concatenate -1 to the version
```

Comment:
It's just weird Openshift client version does have such a suffix in the mirror. Do we have any guidence from Openshift team for this?

Reply:
I'm only guessing but it might be a build number.

Reply:
I just found the version as **"OpenShift Web Console"** version in the `https://[host]:8443/console/about` page

Reply:
Ok, let's merge it. I think it's more temporary solution. We'll see.